cantools/database/can/message.py

```diff
@@ -55,7 +55,19 @@ def __init__(self,
         self._length = length
         self._signals = signals
         self._signals.sort(key=start_bit)
-        self._comment = comment
+
+        # if the 'comments' argument is a string, we assume that is an
+        # english comment. this is slightly hacky because the
+        # function's behavior depends on the type of the passed
+        # argument, but it is quite convenient...
+        self._comments = {}
+        if isinstance(comments, str):
+            # use the first comment in the dictionary as "The" comment
+            self._comments = { "EN": comments }
```

Comment:
Hmm... As the language of the comment is unknown it's probably better to do `self._comments = {None: comments}`.

Reply:
changed. Thanks!